packages/smart-wallet/src/smartWallet.js

```diff
@@ -446,10 +469,26 @@ export const prepareSmartWallet = (baggage, shared) => {
             },
             onStatusChange: offerStatus => {
               logger.info('offerStatus', offerStatus);
+
               updatePublishKit.publisher.publish({
                 updated: 'offerStatus',
                 status: offerStatus,
               });
+
+              const isOfferPossiblyExitable = !(
```

Comment:
there's got to be a better name for this state. "unresolved"?   @Chris-Hibbert  or @dckc may have ideas

Reply:
I looked in our [UserSeat](https://docs.agoric.com/reference/zoe-api/zoe.html#userseat-object) docs, and I'm still not clear on all the possible states and transitions, let alone good names for them.

"pending" is the first word that came to mind, but I don't know whether it's accurate.

Reply:
I'd offer 'live' or 'open'. The alternative is `exited`.